src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -135,6 +136,9 @@ public void run() {
 				CommandsFeature.setupCommandManager();
 				MacroFeature.setupMacroDriver();
 
+				CommandFactory.getInstance();
```

Comment:
Illogical line. Change the design so it is easy to understand (clean code is self expaining)